hw_6n.py

```diff
@@ -0,0 +1,33 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів
+# add: додати запис
+# delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі
+# show <name>: детальна інформація по імені
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+
+phone_book = {'Vadym':'0665699589'}
+while True:
+    enter = input('''You can see all contacts(stats), all names(list), someone\'s number(show), (add) or (delete) contacts. 
+    Enter your comand: ''')
+    if enter == 'add':
+        name = input('Enter name to add: ')
+        phone = input('Enter phone to add: ')
+        phone_book.setdefault(name, phone)
```

Comment:
You should check if name already exists before doing this action since it is not allowed to override records according to the task

Reply:
The setdefault() method returns the value of the item with the specified key.
If the key does not exist, insert the key, with the specified value. So there is no need to additional check because you can't overwrite existing name. What you think?

Reply:
You are right